### DISCUSSION TITLE
Error when connect to a Cubbit S3 space

### DISCUSSION THREAD

**Discussion by alex-amclu** [iterate-ch/cyberduck#16083] [Q&A] [answered]
Good morning, for a few days now when I try to connect to Cubbit's S3 space, mountainduck reports the error 

```
Failed to parse XML document with header class . . . . 
```

By entering the name of the bucket in the profile configuration, the connection works but mountainduck does not synchronize the modified files and asks to disconnect the cloud.
Using WinScp the two-way communication works fine.
We use mountain duck version 4.16.00 on Windows 10 64bit ver. 22H2.

**Comment by sMacchianera** [comment]
It seems something is wrong with cubbit, you need to specify the bucket path and you can no longer access the root. I have the same problem too

**Comment by dkocher** [comment, marked as answer]
The problem is that the Cubbit S3 services returns the wrong content type in the response listing all buckets and is thus not S3 interoperable. 

```
Expected XML document response from S3 but received content type text/plain; charset=utf-8.
```

#16085 will resolve the failure to display the original error.
